src/Sulu/Bundle/SecurityBundle/UserManager/UserManager.php

```diff
@@ -685,7 +685,9 @@ private function processEmail(UserInterface $user, $email, $contact = null)
                 $user->setEmail($email);
             }
         } else {
-            if ($email && 0 !== \strcasecmp($email, $user->getEmail()) && !$this->isEmailUnique($email)) {
+            $emailChanged = $email && (null === $user->getEmail() || 0 !== \strcasecmp($email, $user->getEmail()));
+
+            if ($emailChanged && !$this->isEmailUnique($email)) {
```

Comment:
Would avoid the duplicated email getEmail call and just go with `??`:

```suggestion
            $currentEmail = $user->getEmail() ?? '';
            if ($email && 0 !== \strcasecmp($email, $currentEmail) && !$this->isEmailUnique($email)) {
```

Reply:
I have added the `$emailChanged` variable, because for me was not clear what `$email && 0 !== \strcasecmp($email, $currentEmail)` means at first glance.

Second point: `$user->getEmail() ?? ''` works, but it is not very explicit.